File: QViz.NET/Periscope.QViz/JSON/AccessUser.cs
namespace Periscope.QViz.JSON
{
	/// <summary>
	/// JSON Model for User Access Details
	/// </summary>
	public class AccessUser
	{
		/// <summary>
		/// OAuth 2.0 Bearer Token of the Authentication
		/// </summary>
		public string access_token;

		/// <summary>
		/// Identifier of the User
		/// </summary>
		public string userId;

		/// <summary>
		/// Unique Name of the User
		/// </summary>
		public string username;

		/// <summary>
		/// First Name of the User
		/// </summary>
		public string firstname;

		/// <summary>
		/// Last Name of the User
		/// </summary>
		public string lastname;

		/// <summary>
		/// Default Constructor
		/// </summary>
		public AccessUser()
		{
			this.access_token = "";
			this.userId = "";
			this.username = "";
			this.firstname = "";
			this.lastname = "";
		}
	}
}

File: QViz.Java/src/main/java/com/periscope/qviz/json/AccessUser.java
package com.periscope.qviz.json;

/**
 * JSON Model for User Access Details
 */
public class AccessUser {

	/**
	 * OAuth 2.0 Bearer Token of the Authentication
	 */
	public String access_token;

	/**
	 * Identifier of the User
	 */
	public String userId;

	/**
	 * Unique Name of the User
	 */
	public String username;

	/**
	 * First Name of the User
	 */
	public String firstname;

	/**
	 * Last Name of the User
	 */
	public String lastname;

	/**
	 * Public key
	 */
	public String qvizPublicKey;

	/**
	 * IS this user part of admin group
	 */
	public boolean isInAdminGroup;

	/**
	 * Default Constructor
	 */
	public AccessUser() {
		this.access_token = "";
		this.qvizPublicKey = "";
		this.userId = "";
		this.username = "";
		this.firstname = "";
		this.lastname = "";
		this.isInAdminGroup = true;
	}

}
